calculator.rb

```diff
@@ -14,6 +16,14 @@ def divide(operand_1, operand_2)
   return operand_1 / operand_2
 end
 
+def exponent(operand_1, operand_2)
+  return operand_1 ** operand_2
+end
+
+def root(operand_1)
```

Comment:
Should this be called `sqrt`?

Reply:
And since there's only one operand, it could just be named `operand`

Reply:
I need it to be operand_1 so that we have minimal variables. I tried just using operand and the operation was undefined.